microsoft365/credential/tool.gpt

```diff
@@ -1,18 +1,25 @@
-Name: Outlook Calendar OAuth Credential
-Share Credential: outlook-cal-cred as outlook.calendar.write
+Name: Microsoft365 OAuth Credential
+Share Credential: m365-cred as microsoft365.write
```

Comment:
My only concern here is that we are effectively deleting everyone's credential with this. Is that worth a standup discussion?